man/runc-restore.8.md

```diff
@@ -74,6 +74,15 @@ daemon. See [criu --lazy-pages option](https://criu.org/CLI/opt/--lazy-pages).
 : Specify an LSM profile to be used during restore. Here _type_ can either be
 **apparamor** or **selinux**, and _label_ is a valid LSM label. For example,
 **--lsm-profile "selinux:system_u:system_r:container_t:s0:c82,c137"**.
+By default, the checkpointed LSM profile is used upon restore.
+
+**--lsm-mount-context** _context_
+: Specify an LSM mount context to be used during restore. Only mounts with an
+existing context will have their context replaced. With this option it is
+possible to change SELinux mount options. Instead of mounting with the
+checkpointed context the mount context will be replaced with the value specified
+using **--lsm-mount-context**.
```

Comment:
It's weird that the option being described refers to itself.

I think it makes more sense to say something like
```suggestion
checkpointed context, the _context_ will be used for mounting.